### ISSUE TITLE
Arrow keys don't work properly with slider

### ISSUE THREAD

**Issue by AurelienSciarra** [pyviz/panel#534]
When I initialize a slider with a certain bound, and then if I change the bounds . I won't be able to move the slider futher than the first bound. But I can pass thoses bounds by moving the slider with my mouse.

Here is the code  

```
import panel as pn

slider=pn.widgets.IntSlider(end=4,start=0)

slider

slider.end=10

```
This issue occurs with all Slider (IntSlider , FloatSlider, DateSlider)

**Comment by philippjfr** [comment]
Seems like a bug in bokeh and should be filed there.

**Comment by AurelienSciarra** [comment]
bug resolved in bokeh 1.3.4
